README

```diff
@@ -1,29 +1,34 @@
-
-SQUID Web Proxy Cache                         http://www.squid-cache.org/
--------------------------------------------------------------------------
+SQUID Web Proxy Cache                        http://www.squid-cache.org/
```

Comment:
The two changed lines were exceeding the 72-character limit (common for plain texts).

I will undo this change if you insist that these two lines must be 73 characters long.